api/user.go

```diff
@@ -206,11 +206,6 @@ func attachDeviceId(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if !(strings.HasPrefix(deviceId, model.PUSH_NOTIFY_APPLE+":") || strings.HasPrefix(deviceId, model.PUSH_NOTIFY_ANDROID+":")) {
```

Comment:
I removed it since it's just pass-thru.  I'll add a check to the push proxy.  This should make adding new ones or customer custom ones easier.  I'll leave the extra validation off.